File: backend/Domain/Entities/CustomerBox.cs
namespace Domain.Entities
{
    public class CustomerBox : BaseEntity<Guid>
    {
        public Guid CustomerId { get; set; }
        public virtual Customer Customer { get; set; }
        public string Label { get; set; }
        public Guid StorageAreaId { get; set; }
        public virtual StorageArea StorageArea { get; set; }
        public Guid BoxStatusId { get; set; }
        public virtual BoxStatus BoxStatus { get; set; }
    }
}

File: frontend/front-desk-app/src/app/models/customerBox.ts
import { ICustomer } from "./customer";
import { IStorageArea } from "./storageArea";
import { IBoxStatus } from "./boxStatus";
import { IBaseEntity } from "./baseEntity";

export interface ICustomerBox extends IBaseEntity<string> {
    customerId: string;
    customer: ICustomer;
    label: string;
    storageAreaId: string;
    storageArea: IStorageArea;
    boxStatusId: string;
    boxStatus: IBoxStatus;
}